GrafanaUI: Added spacing token to Divider Component (#72725)
grafana/grafana: packages/grafana-ui/src/components/Divider/Divider.mdx

```

### Modifiers

> For the margin to be defined can use the spacing prop and pass a spacing token into it.

Reviewer: ```suggestion
Pass a spacing token into the `spacing` prop to adjust the margin.
```
